Utils/PluginActivator/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+
+# If not stated otherwise in this file or this component's license file the
+# following copyright and licenses apply:
+#
+# Copyright 2020 Metrological
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required(VERSION 3.10.3)
+
+project( PluginActivator )
+
+set( CMAKE_CXX_STANDARD 14 )
+set( CMAKE_CXX_STANDARD_REQUIRED ON )
+
+add_executable(${PROJECT_NAME}
+    source/Module.cpp
+    source/main.cpp
+    source/COMRPCStarter.cpp
+)
+
+target_compile_options(${PROJECT_NAME}
+    PRIVATE
+    -Wall
```

Comment:
I am not the CMake expert but I think we typically drag on the Compiler Settings through some CMake contrsuctio, taggin @bramoosterhuis  to eleborate. This CMake stuff should enforce the strictest warning level (and set C++11). It also, for example, set the wchar/char usage within Thunder.

Reply:
Switched to `CompileSettingsDebug::CompileSettingsDebug` instead of adding `-Wall` manually - let me know if there's a better way to do this